libpod/image/pull.go

```diff
@@ -263,7 +263,7 @@ func (ir *Runtime) doPullImage(ctx context.Context, sc *types.SystemContext, goa
 		copyOptions.SourceCtx.SystemRegistriesConfPath = systemRegistriesConfPath // FIXME: Set this more globally.  Probably no reason not to have it in every types.SystemContext, and to compute the value just once in one place.
 		// Print the following statement only when pulling from a docker or atomic registry
 		if writer != nil && (imageInfo.srcRef.Transport().Name() == DockerTransport || imageInfo.srcRef.Transport().Name() == AtomicTransport) {
-			io.WriteString(writer, fmt.Sprintf("Trying to pull %s...", imageInfo.image))
+			_, _ = io.WriteString(writer, fmt.Sprintf("Trying to pull %s...", imageInfo.image))
```

Comment:
I think we can error out here. If `writer` went south, we will error out later in any case but we'd burn some cycles until then.